packages/consumption/src/modules/requests/outgoing/OutgoingRequestsController.ts

```diff
@@ -51,8 +51,17 @@ export class OutgoingRequestsController extends ConsumptionBaseController {
         const parsedParams = CanCreateOutgoingRequestParameters.from(params);
         if (parsedParams.peer) {
             const relationship = await this.relationshipResolver.getRelationshipToIdentity(parsedParams.peer);
-            if (!relationship || !(relationship.status === RelationshipStatus.Pending || relationship.status === RelationshipStatus.Active)) {
-                return ValidationResult.error(CoreErrors.requests.noMatchingRelationship(parsedParams.peer.toString()));
+            if (!relationship) {
+                return ValidationResult.error(
+                    CoreErrors.requests.missingRelationship(`You cannot create a request to ${parsedParams.peer.toString()} since you are not in a relationship.`)
+                );
+            }
+            if (!(relationship.status === RelationshipStatus.Pending || relationship.status === RelationshipStatus.Active)) {
+                return ValidationResult.error(
+                    CoreErrors.requests.wrongRelationshipStatus(
+                        `You cannot create a request to ${parsedParams.peer.toString()} since the relationship is in status ${relationship.status}.`
```

Comment:
```suggestion
                        `You cannot create a request to ${parsedParams.peer.toString()} since the relationship is in status '${relationship.status}'.`
```

Reply:
same things as in Incoming